docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+# User Guide to Integrate a New Platform into the OpenROAD Flow
+
+# Table of Contents <a name="TOC"></a>
+
+- [Table of Contents](#TOC)
+- [Overview](#Overview)
+- [Prerequisites](#Prerequisites)
+- [Adding a new Platform to OpenROAD](#APOpenROAD)
+	- [Setup](#Setup)
+		- [Makefile](#Makefile)
+		- [Platform Directory](#PlatformDirectory)
+		- [Design Directory](#DesignDirectory)
+	- [Platform Configuration](#PlatformConfiguration)
+	- [Design Configuration](#DesignConfiguration)
+		- [config.mk](#configmk)
+		- [constraint.sdc](#constraintsdc)
+		- [Liberty, LEF, and GDS Files](#LibertyLEFGDSFiles)
+	- [Behavioral Models](#BehavioralModels)
+		- [Gated Clock](#GatedClock)
+		- [Latch](#Latch)
+	- [FastRoute Configuration](#FastRouteConfiguration)
+	- [Metal Tracks Configuration](#MetalTracksConfiguration)
+	- [PDN Configuration](#PDNConfiguration)
+	- [Tapcell Configuration](#TapcellConfiguration)
+	- [setRC Configuration](#setRCConfiguration)
```

Comment:
Shouldn't there be a section for ICeWall as well?

Reply:
IceWall is not mandatory since it is only needed for IO Pads, pad frames etc.